From 0b9328300f14d1a145b6e04e47e51cbed95d1dd1 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Fri, 17 Aug 2018 16:53:10 +0800
Subject: [PATCH 4495/5242] MLK-19258-4 ARM64: dts: fsl-imx8mq.dtsi: enable
 USB2.1 LPM support

commit  c6fc74942fb9afb0a1acf2c6fea92628ab00eebb from
https://source.codeaurora.org/external/imx/linux-imx.git

With these two properties enabled, the dwc3 driver at 850D can ACk
LPM packet from host, and the later suspend/resume signals are correct.

Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index 4c5d9dc..5ba0009 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -956,6 +956,8 @@
 			snps,power-down-scale = <2>;
 			usb3-resume-missing-cas;
 			usb3-lpm-capable;
+			snps,has-lpm-erratum;
+			snps,lpm-nyet-threshold = <0xf>;
 			status = "disabled";
 		};
 	};
-- 
1.7.9.5

